src/components/10-atoms/checkbox/story.js

```diff
@@ -20,25 +20,36 @@ storyCheckbox.add('Checkbox', () => {
   const refId = text('refId', `checkbox-${createRefId()}`);
   const label = text('label', 'I agree to conditions of data protection.');
   const name = text('name', 'my-checkbox');
-  const variant = select('variant', ['square', 'checkmark']);
+  const variant = select('variant', [
+    'square',
+    'checkmark',
+    // 'inverted-square', // not official supported yet
+    'inverted-checkmark',
+  ]);
   const checked = boolean('checked', true);
   const disabled = boolean('disabled', false);
   const errortext = boolean('error', false);
   const required = boolean('required', false);
 
   const wrapper = document.createElement('div');
   const template = html`
-    <axa-checkbox
-      refId="${refId}"
-      class="hover"
-      name="${name}"
-      variant="${variant}"
-      label="${label}"
-      ?disabled="${disabled}"
-      ?checked="${checked}"
-      ?required="${required}"
-      error="${errortext ? 'Please accept our terms and conditions.' : ''}"
-    ></axa-checkbox>
+    <div
+      style="${variant && variant.indexOf('inverted') > -1
```

Comment:
nicer to define `isInverted` outside the template

Reply:
That makes it more complex. Dont change that.